### PR TITLE
Update staging instructions to use node_util.py and ufw configuration

### PR DESCRIPTION
**Pull request by cryofracture**
Closed original PR and submitting under new branch and changes as laid out in the contribution guidelines. Docs have been rebuilt with `build.sh` and should reflect the same changes as previously. 

Documentation now will stage all protocol upgrades for the casper network designated, and provides a section explaining how to set up the node's firewall if desired.

### CODE REVIEW

File: src/ubuntu/stage-upgrade-mainnet.md [hunk 1]
```diff
@@ -1,105 +1,4 @@
 ### Stage the upgrades
```
Comment (mrkara) [inline src/ubuntu/stage-upgrade-mainnet.md:1]: This seems redundant now. I think either this section or the previous subtitle `Stage all protocol upgrades` needs to be removed.

File: src/ubuntu/stage-upgrade.md [hunk 2]
```diff
@@ -1,137 +1,4 @@
 ### Stage the upgrades
```
Comment (mrkara) [inline src/ubuntu/stage-upgrade.md:1]: This seems redundant now. I think either this section or the previous subtitle `Stage all protocol upgrades` needs to be removed.

**Review by mrkara — CHANGES_REQUESTED** [review]

**Comment by mrkara** [comment]
@cryofracture Thanks a lot for the new PR. Could you please take a look at the comments above?

File: src/ubuntu/stage-upgrade-mainnet.md (hunk 1 above)
Comment (cryofracture) [reply src/ubuntu/stage-upgrade-mainnet.md:1]: Removed staging-upgrade-mainnet.md from the repo and removed the include statement in the base readme.

File: src/ubuntu/stage-upgrade.md (hunk 2 above)
Comment (cryofracture) [reply src/ubuntu/stage-upgrade.md:1]: Removed from `setup-testnet-validator-from-scratch.md` and deleted the file for now.

**Comment by cryofracture** [comment]
@mrkara Files have been removed now - please review again.

**Comment by mrkara** [comment]
@cryofracture Thanks a lot for your contribution!

**Pull request merged**